nikola/data/samplesite/pages/dr-nikolas-vendetta.rst

```diff
@@ -360,7 +360,7 @@ once more to its former business-like tone.
 "I am not going to tell you very much about it. But from what I do tell
 you, you will be able to gather a great deal and imagine the rest. To
 begin with, there is a man living in this world to-day who has done me a
-great and lasting injury. What that injury is is no concern of yours.
+great and lasting injury. What that injury is no concern of yours.
```

Comment:
It’s a book from Project Gutenberg, there’s no point in modifying it here, and this typo might exist in the original book.

Reply:
And actually, this is *not* a typo.

Reply:
ok